other-file.txt

```diff
@@ -3,4 +3,4 @@ Congrats, you found the OTHER file!
 There's another question in this one, but I promise it's not too tricky!
 
 Question: What is your favourite song?
-Your answer:
\ No newline at end of file
+Your answer: Dzigbordi by Lord Paper (A Ghanaian song)
```

Comment:
will need a interpreter but am sure it will be a good choice

Reply:
@sobiawaqar It's a love song about a guy who is in love with a girl but her parents do not accept him because of what tribe he is from which he doesn't agree with